README.md

```diff
@@ -43,3 +43,6 @@
   ```
   npm run test
   ```
+
+## Important
+Since travis is failing on executing a11y tests due to out of date suid you need to execute `npm run a11y` locally before sending any pr to verify we're following accessibility standars.
```

Comment:
What do you think of adding `npm run a11y` in `lint-staged`?

Reply:
I think `lint-staged` is fine for linting and format stuff, but if we add `ally` we force each commit yo be accessible, which I think is not a good idea while developing.

Reply:
Yeah, that might have gone bad, I had not thought about that part 😅.